ejb/ejb2-connector/ejb2-connector-1.0.0/org.wso2.carbon.connector/src/test/java/org/wso2/carbon/connector/integration/test/ejb2/checkStateless.java

```diff
@@ -0,0 +1,35 @@
+package org.wso2.carbon.connector.integration.test.ejb2;/*
+* Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import com.ejb2connector.stateless.Hello;
+import com.ejb2connector.stateless.HelloHome;
+
+import javax.ejb.CreateException;
+import javax.naming.Context;
+import javax.naming.NamingException;
+import java.rmi.RemoteException;
+
+public class checkStateless {
```

Comment:
Fixed in public branch https://github.com/wso2-extensions/esb-connector-ejb2.0/tree/master/src/test/java/org/wso2/carbon/connector/integration/test/ejb2